mirage/config.ts

```diff
@@ -420,4 +421,14 @@ export default function(this: Server) {
 
     // node analytics
     this.get('/metrics/query/node_analytics/:nodeID/:timespan', getNodeAnalytics);
+
+    // Addon service
+    this.urlPrefix = addonServiceUrl;
+    this.namespace = '/v1';
```

Comment:
Should we get this from `app/adapters/addon-service.ts`?

Reply:
I like this idea. Updating now